CONTRIBUTING.md

```diff
@@ -162,162 +268,115 @@ fixes:
 
-### Test
+To check what the rendered html output of the API documentation, tutorials,
+and release notes will look like for the current state of the repo, run:
 
-Once you've made a code change, it is important to verify that your change
-does not break any existing tests and that any new tests that you've added
-also run successfully. Before you open a new pull request for your change,
-you'll want to run the test suite locally.
+    tox -edocs
+    
+This will build all the documentation into `docs/_build/html`. The main page `index.html`
+will link to the relevant pages in the subdirectories, or you can navigate manually:
 
-The easiest way to run the test suite is to use
-[**tox**](https://tox.readthedocs.io/en/latest/#). You can install tox
-with pip: `pip install -U tox`. Tox provides several advantages, but the
-biggest one is that it builds an isolated virtualenv for running tests. This
-means it does not pollute your system python when running. Additionally, the
-environment that tox sets up matches the CI environment more closely and it
-runs the tests in parallel (resulting in much faster execution). To run tests
-on all installed supported python versions and lint/style checks you can simply
-run `tox`. Or if you just want to run the tests once run for a specific python
-version: `tox -epy37` (or replace py37 with the python version you want to use,
-py35 or py36).
+* `apidocs/`:  Contains the API docs automatically compiled from module docstrings.
+* `tutorials/`: Contains the `.rst` tutorials with automatically executed Jupyter cells.
+* `release_notes.html`: Contains the release notes.
 
-If you just want to run a subset of tests you can pass a selection regex to
-the test runner. For example, if you want to run all tests that have "dag" in
-the test id you can run: `tox -epy37 -- dag`. You can pass arguments directly to
-the test runner after the bare `--`. To see all the options on test selection
-you can refer to the stestr manual:
-https://stestr.readthedocs.io/en/stable/MANUAL.html#test-selection
+To build release notes and API docs without building the Jupyter cells in the `.rst` files under `tutorials/`, 
+which is a relatively slow process, you can run `tox -edocsnorst` instead.
 
-If you want to run a single test module, test class, or individual test method
-you can do this faster with the `-n`/`--no-discover` option. For example:
 
-to run a module:
-```
-tox -epy37 -- -n test.python.test_examples
-```
-or to run the same module by path:
+### Adding deprecation warnings
+Qiskit Experiments is part of Qiskit and, therefore, the [Qiskit Deprecation Policy](https://qiskit.org/documentation/contributing_to_qiskit.html#deprecation-policy) fully applies here. We have a deprecation decorator for showing deprecation warnings. To deprecate a function:
```

Comment:
When @jakelishman's changes get merged in to the meta package I think the link to the deprecation policy will change. Either we can wait to merge this PR until after Jakes work is merged, or just remember to update the link at a later date